templates/chatbot/content/Containerfile

```diff
@@ -0,0 +1,8 @@
+FROM registry.access.redhat.com/ubi9/python-311:1-62.1716478620
```

Comment:
IMO, we should not checkin chatbot and codegen on this repo.

We will have 2 copies to maintain. Since codegen and chatbot were the first 2, its ok to have them in the main repo and not here

Reply:
If we don't want to have chatbot/codegen in this repo then what samples will the experiemental local deploy reference?

This duplication issue would eventually be remedied by https://issues.redhat.com/browse/DEVHAS-723 I believe where we take a url for the sample

Reply:
Ah you are right